backend/src/main/java/com/gildedrose/ItemsPresenter.java

```diff
@@ -10,21 +10,27 @@ public String singleItemJson(Ageable item) {
 
         return String.format("""
                 {
+                    "id": "%s",
                     "name": "%s",
-                    "price": "%f"
+                    "quality": "%s",
+                    "sellIn": "%s",
+                    "price": "%.2f"
                 }
-                """, item.getName(), item.getPrice());
+                """, item.getId(), item.getName(), item.getQuality(), item.getSellIn(), item.getPrice());
     }
 
     public String allItemsJson(Collection<Ageable> items) {
         StringBuilder allItemsJson = new StringBuilder();
         for (Ageable item : items) {
             allItemsJson.append(String.format("""
```

Comment:
Suggestion: You can probbly re-use `singleItemJson` here like:
```Java
...
allItemJson.append(singleItemJson(item));
```

Reply:
That ended up breaking the tests. In allItemJson, there is a "," at the end of the bracket (see screenshot) this comma is used as the index to place the final closing bracket.

There was a workaround to do this but it seems more confusing than just duplicating the string literal.
```
 for (Ageable item : items) {
            allItemsJson.append(singleItemJson(item)).insert(allItemsJson.length() - 1, ",");
        }
```
**Screenshot of allItemsJson**
<img width="849" alt="Screen Shot 2022-08-13 at 2 38 26 PM" src="https://user-images.githubusercontent.com/61036790/184508707-82a226be-ba03-4c28-a213-bc26259b8c08.png">

Reply:
Since we're using StringBuilder, can't we just do:
```Java
...
allItemJson.append(singleItemJson(item));
allItemJson.append(",");
```
I'm also wondering if we can use the builtin json lib to build a json object instead.

Reply:
I like the idea of a builtin json lib. We'll look into Jackson